SMLHelper/Patchers/CraftDataPatcher.cs

```diff
@@ -11,6 +11,8 @@ internal class CraftDataPatcher
 
         private static readonly Func<TechType, string> AsStringFunction = (t) => t.AsString();
 
+#if SUBNAUTICA
```

Comment:
Split `CraftDataPatcher.cs` into multiple files for SN and BZ, this is getting messy.

Reply:
This will be handled by @ahk1221 and @MrPurple6411